health-services/project-factory/src/server/utils/excelUtils.ts

```
@@ -399,5 +445,265 @@
+
+  logger.info("Worksheet values updated successfully for employees.");
+}
+
+function addHeadersAndFormatForCampaignEmployees(
+  userWorkSheet: any,
+  headerRow: any,
+  emptyColumnIndex: number,
+  newHeaderValues: string[]
+) {
+  if (emptyColumnIndex === -1) {
+    emptyColumnIndex = headerRow.values.length;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)